UPDATING.md

```diff
@@ -23,6 +23,12 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Operator return values are not pushed to XCom by default
```

Comment:
Can we make a note of this here as well -> https://github.com/apache/airflow/blob/master/docs/concepts.rst#xcoms